Sources/NonEmpty/NonEmpty+Random.swift

```diff
@@ -5,7 +5,12 @@ extension NonEmpty {
   }
 
   public func randomElement() -> Element {
+    #if swift(>=4.2)
+    var generator = SystemRandomNumberGenerator()
```

Comment:
Don't have time to check right now, but did this get back-ported to 4.1.x compatibility mode in the latest Xcode?